locations/jclouds/src/main/java/brooklyn/location/jclouds/JcloudsLocation.java

```diff
@@ -622,6 +624,14 @@ protected JcloudsSshMachineLocation obtainOnce(ConfigBag setup) throws NoMachine
             if (node == null)
                 throw new IllegalStateException("No nodes returned by jclouds create-nodes in " + setup.getDescription());
 
+            boolean windows = node.getOperatingSystem().getFamily().equals(OsFamily.WINDOWS);
+
+            if (windows) {
+                // FIXME: Tidy this up and allow for user-specified credentials
```

Comment:
Am I reading this right, that we currently don't support creating a new user or support setting the existing (e.g. administrator) password?

If so, we should include that in the list of things to do for being ready for production.

How does the Brooklyn user discover the user/password of their VM for if they want to subsequently log in? I presume that won't be displayed in the Brooklyn web console at all currently?